discovery-provider/plugins/notifications/src/main.ts

```diff
@@ -93,6 +95,14 @@ export class Processor {
     return Boolean(isEnabled)
   }
 
+  getIsBrowserPushEnabled(): boolean {
```

Comment:
querying the feature flag here would require the plugin to restart to read any change -- is this flag pretty much fully rolled out?

Reply:
I believe RemoteConfig refreshes itself under the hood, it'll also get recalled every time the isRunning loop runs. Flag is already turned on for stage. That's what fixed Reed and Marcus' issue earlier today when they weren't seeing web push for other notif types.